.bumpversion.cfg

```diff
@@ -2,7 +2,7 @@
 commit = True
 tag = False
 message = Bump version: {current_version} → {new_version}
-current_version = 1.5.1
+current_version = 1.6.3
```

Comment:
Please do not change .bumpversion.cfg.
This file is used to automatically update version information when releasing. 
Please don't modify the version in setup.py too. 

-> This part will be explained further when other corrections are completed.